generators/app/templates/Procfile.dev

```diff
@@ -1,2 +1,2 @@
-server: node express.js
-client: rails s -p 3000
+server: cd server && rails s -p 3000
+client: cd client && npm start
```

Comment:
Well, I used node-foreman before and it worked beautifully. What kind of problem did you have?